hedera-node/test-clients/src/main/resource/bootstrap.properties

```diff
@@ -226,3 +226,5 @@ virtualdatasource.jasperdbToMerkledb=true
 staking.sumOfConsensusWeights=500
 cache.cryptoTransfer.warmThreads=30
 hedera.config.version=0
+staking.maxStakeRewarded=5000000000000000000
+staking.rewardBalanceThreshold=0
```

Comment:
These won't actually get picked up by the modular tests, but if we need them for the mono tests, then that's cool.

Reply:
It seems they are getting picked up @rbair23 . Without these two there is an error while running tests like this
```
2023-08-09 08:49:52.639 ERROR  71   StakingPeriodTimeHook - CATASTROPHIC failure updating end-of-day stakes
java.lang.IllegalStateException: 'bootstrap.properties' is missing properties: [staking.maxStakeRewarded, staking.rewardBalanceThreshold]!
	
```